docs/fx-core/spfx-help.md

```diff
@@ -0,0 +1,66 @@
+# SPFx prerequisites troubleshoot
+
+## Overview
+
+For SPFx app, Teams Toolkit uses Yeoman Generator for scaffolding. This requires both [Yeoman Generator](https://yeoman.io/) and the correct SPFx generator version to be installed.
```

Comment:
Yeoman generator means the related generator for scaffolding, maybe use 'Yeoman CLI' here?